remove paths as they are invalid in the default chroot

Index: conf/config.inc.php
--- conf/config.inc.php-dist.orig
+++ conf/config.inc.php-dist
@@ -32,8 +32,8 @@
 
 	// Specify the path to the database dump utilities for this server.
 	// You can set these to '' if no dumper is available.
-	$conf['servers'][0]['pg_dump_path'] = '/usr/bin/pg_dump';
-	$conf['servers'][0]['pg_dumpall_path'] = '/usr/bin/pg_dumpall';
+	$conf['servers'][0]['pg_dump_path'] = '';
+	$conf['servers'][0]['pg_dumpall_path'] = '';
 
 	// Example for a second server (PostgreSQL for Windows)
 	//$conf['servers'][1]['desc'] = 'Test Server';
